### DISCUSSION TITLE
Main or Master

### DISCUSSION THREAD

**Discussion by Tonterias** [community/community#70241] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

It is just me or when you say main, you really want to say master?

git remote add origin https://github.com/Tonterias/testunify.git
git branch -M **main**
git push -u origin **main**

**Comment by ghostinhershell** [comment]
Hey @Tonterias 👋🏾 

Thanks so much for posting in the community. I personally prefer `main` over `master`. Although the default branch name for new repositories is `main`, you can set another default. There are a couple of paths you can take to do so depending on your circumstances:

- For users, you can make that update at https://github.com/settings/repositories page
- For organization owners, you would check out https://github.com/organizations/YOUR-ORGANIZATION/settings/repository-defaults page
- For enterprise administrators, check out https://github.com/enterprises/YOUR-ENTERPRISE/settings/member_privileges page

^Note that, for the last 2 links there, you will need to enter either your organization or your enterprise slug into the URL in order for that URL to be valid. 

Hope that helps!
